metomi/rosie/suite_id.py

```diff
@@ -456,6 +404,78 @@ def _from_location(self, location):
         if "commit:revision" in info_entry:
             self.revision = info_entry["commit:revision"]
 
+    @staticmethod
+    def _find_vc_file_from_location(location):
+        """Search a location and parents for a version control log file.
+
+        Args:
+            location: Path to search.
+
+        Returns:
+            If a file is found, a path, else None.
+        """
+        suite_engine_proc = SuiteEngineProcessor.get_processor()
+        suite_dir_rel_root = getattr(
+            suite_engine_proc, "SUITE_DIR_REL_ROOT", None
+        )
+        loc = Path(location).expanduser().resolve()
+        if suite_dir_rel_root:
+            sdrr = Path('~', suite_dir_rel_root).expanduser().resolve()
+        try:
+            loc.relative_to(sdrr)
```

Comment:
Think you will get an sdrr is not defined if `suite_dir_..` is None here. Could indent block into the `if` to fix.

Reply:
Yes.